load-generator/src/main/java/com/amazon/opentelemetry/load/generator/emitter/OtlpMetricEmitter.java

```diff
@@ -53,13 +56,37 @@ public void emitDataLoad() throws Exception {
 
   @Override
   public void nextDataPoint() {
-    apiBytesSentCounter
-            .add(10, METRIC_ATTRIBUTES);
-    mutateApiLatency();
+    log.info("Updating metrics for : {}" , param.getMetricType());
```

Comment:
Should this be debug rather than info?  Seems like it could generate a lot of log noise.

Reply:
Thanks. Made changes accordingly.